tests/func/test_init.py

```diff
@@ -64,3 +68,37 @@ def test_cli(self):
         self.assertEqual(ret, 0)
 
         self._test_init()
+
+
+class TestInitQuietShouldNotDisplayWelcomeScreen(TestGit):
```

Comment:
It's better to not add TestUnit-style tests anymore. In pytest style this will look like:
```python
def test_init_quiet_should_not_display_welcome_screen(repo_dir, caplog):
    # ...

    assert "What's next" not in caplog.text
```
You can look up our fixtures like `repo_dir` in `tests/conftest.py`.